oci/config/convert/runtime.go

```diff
@@ -148,11 +147,11 @@ func MutateRuntimeSpec(g rgen.Generator, rootfs string, image ispec.Image) error
 	}
 
 	// Set optional fields
-	var ports []string
-	for port := range image.Config.ExposedPorts {
-		ports = append(ports, port)
+	ug, err := ugen.NewFromImage(image)
+	if err != nil {
```

Comment:
Currently, `NewFromImage()` always returns `nil`. Is it meant to be safe for future changes or could it be changed?